lib/search.js

```diff
@@ -17,28 +17,47 @@ module.exports = new class LibSearch {
     }
 
     optimizeQueryTerms(input) {
+        // parse
+        let normalize = this.normalizeProt();
+        input.imdbid = input.imdbid && parseInt(input.imdbid.toString().replace('tt', ''), 10)
+        input.season = input.season && parseInt(input.season)

```

Comment:
If the season property exists, convert it to integer, otherwise leave it undefined, the same applies to all other properties, the key is in the && operator